TestFramework/TestSupportFiles/ToDo.cs

```diff
@@ -17,6 +17,9 @@ public class ToDo : Entity
 
 		[JsonProperty("value")]
 		public int Value { get; set; }
+
+		[JsonProperty("bool_value")]
+		public bool BoolVal { get; set; }
```

Comment:
@tejasranade We do not currently have a test that covers `System.Boolean`, but this is equivalent to `bool`.  In fact, the IDE will throw a warning stating that `System.Boolean` can be simplified to `bool`, and so I don't believe that a specific test for the `System.Boolean` type is strictly necessary.